controller/users.go

```diff
@@ -167,6 +167,15 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 			keycloakUserProfile.Username = updatedUserName
 		}
 
+		updatedRegistratedCompleted := ctx.Payload.Data.Attributes.RegistrationCompleted
+		if updatedRegistratedCompleted != nil {
+			if *updatedRegistratedCompleted == false {
+				jerrors, _ := jsonapi.ErrorToJSONAPIErrors(goa.ErrInvalidRequest(fmt.Sprintf("invalid value assigned to registration_completed")))
+				return ctx.BadRequest(jerrors)
```

Comment:
Thanks, I've updated the content inside Sprintf() to have more information, and logged it separately too.

https://github.com/almighty/almighty-core/pull/1233/files#diff-da062b312519ee2a61dd669f1ec964d4R173

I'm keeping this as `ctx.BadRequest(..)` to keep it consistent with other places in this controller method :)